app/src/main/java/com/xmartlabs/taskloans/Config.kt

```diff
@@ -23,9 +23,7 @@ object Config {
 
   const val API_BASE_URL = BuildConfig.API_BASE_URL
 
-  const val URL_SIGN_IN = "signin"
-  const val URL_SIGN_UP = "signup"
-  const val URL_TASKS_USERS = "tasks/1/users"
+  const val TASK_LOANS_ID = "1"
```

Comment:
what do you think about `CLEAN_DISHES_TASK_ID`? I don't remember exactly the task content